lib/bundle/extend/os/linux/skipper.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Bundle
+  module Skipper
+    module_function
+
+    def skip?(entry)
+      return generic_skip unless [:cask, :mas].include?(entry.type)
```

Comment:
I think this will address @issyl0's problem:

```suggestion
      return generic_skip? unless [:cask, :mas].include?(entry.type)
```

Reply:
Sadly not, this fails too with the same `undefined method` error.